test/integration/targets/vault/runme.sh

```diff
@@ -2,6 +2,45 @@
 
 set -eux
 
+MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+trap 'rm -rf "${MYTMPDIR}"' EXIT
+
+# create a test file
+TEST_FILE="${MYTMPDIR}/test_file"
+echo "This is a test file" > "${TEST_FILE}"
+
+
+# encrypt it
+ansible-vault encrypt --vault-password-file vault-password "${TEST_FILE}"
```

Comment:
Script based tests should pass `"$@"` to ansible* commands so that increased verbosity can be used, unless there's a reason why this would interfere with the test. The same applies to the other uses of ansible* commands in this script.